package.json

```diff
@@ -1,4 +1,10 @@
 {
+
+
+"dependencies": {
+		"http2-wrapper": "1.0.0-beta.4.4"
```

Comment:
I would rather not add this dependency and instead let the user choose their library of choice for this. I wouldn't mind adding an example in the README of how to do this.

Reply:
peerDependencies  http2-wrapper